config/index.js

```diff
@@ -541,6 +541,14 @@ var conf = convict({
       }
     }
   },
+  serverVerification: {
+    key: {
+      default: 'YOU MUST CHANGE ME',
+      doc: 'server verification key, as used by content-server',
+      format: String,
+      env: 'SERVER_VERIFICATION_KEY'
```

Comment:
Naming nit: this name is hard to grok without a lot of context on the work that we're doing over in content-server, and "server verification" is vague enough that future readers could easily be mislead about the purpose and scope of this key.  Having clear meaningful names is especially important when the code is part of the security boundary, so I suggest we try to name this after what it *does* rather than after the feature it was added in service of.

From the auth-server's perspective, the purpose of this key is to let other trusted servers specify the originating IP address that we use for security checks.  So as a strawman, I propose naming this more specifically along those lines, such as:

* `ORIGINAL_IP_ADDRESS_SIGNING_KEY` for configuring the secret key
* `originalIPAddress` and `originalIPAddressSignature` for the new request parameters

Or something like that anyway :-).  Don't feel like you have to take my specific suggestion, but please consider a name that will make it obvious to future readers the scope of capabilities managed by this key when viewed in isolation.